docs/source/index.rst

```diff
@@ -41,34 +55,33 @@ The following commands will download and build UCX v1.6 :doc:`release <download>
     $ ./contrib/configure-release --prefix=$PWD/install
     $ make -j8 install
 
+    $ gcc test/examples/ucp_client_server.c -lucp -lucs -o ucp_client_server \
+          -Iinstall/include -Linstall/lib
+    $ export LD_LIBRARY_PATH=$PWD/instal/lib
+    $ ./ucp_client_server &
+    $ ./ucp_client_server -a 2.1.3.20
```

Comment:
s/2.1.3.20/$(hostname)

Reply:
changed to <ip-address>, this test does not support gethostbyname